src/frontend/apps/drive/package.json

```diff
@@ -10,7 +10,7 @@
     "build-theme": "cunningham -g css,scss -o src/styles && mv src/styles/cunningham-tokens.scss src/styles/cunningham-tokens-sass.scss"
   },
   "dependencies": {
-    "@gouvfr-lasuite/ui-kit": "0.4.0",
+    "@gouvfr-lasuite/ui-kit": "0.4.1",
```

Comment:
Display a red overlay + message explaining why you can't drop would be great!

<img width="391" alt="Capture d’écran 2025-04-25 à 14 35 41 (2)" src="https://github.com/user-attachments/assets/b81be6db-be83-41c8-aec0-fe7ac517f2ee" />

Reply:
issue created